### PR TITLE
fix(config): add additional productID for the Yale YKFCON

### PR DESCRIPTION
**Pull request by disforw**

### CODE REVIEW

**Comment by disforw** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x0129/ykfcon.json [hunk 1]
```diff
@@ -8,12 +8,17 @@
...
+		{
+			"productType": "0x0600",
+			"productId": "0x0000"
 		}
 	],
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"supportsZWavePlus": true,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/ykfcon.json]:
```suggestion
```
We don't use this anymore

File: packages/config/config/devices/0x0129/ykfcon.json [hunk 2]
```diff
@@ -29,7 +34,7 @@
 			"$import": "templates/yale_template.json#auto_relock"
 		},
 		"3": {
-			"$import": "templates/yale_template.json#auto_relock_time_180"
+			"$import": "templates/yale_template.json#manual_relock_time"
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/ykfcon.json:36]: Did this change?
/cc @blhoward2

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0129/ykfcon.json (hunk 2 above)
Comment (disforw) [reply packages/config/config/devices/0x0129/ykfcon.json:36]: As per documentation, param 3 is 7-60 with default value of 7. This matches the “manual_relock_time” template.
http://manuals-backend.z-wave.info/make.php?lang=en&sku=YKFCON&cert=ZC10-18015918

**Review by disforw — COMMENTED** [review]
No changes made.

File: packages/config/config/devices/0x0129/ykfcon.json (hunk 2 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0129/ykfcon.json:36]: This looks right but that parameter is inconsistent in the template itself. It should be renamed to Auto Relock Time to match the others and the description should be removed. The template reference itself should also change to match the others (and fixed in any other files).

I can look at this later unless someone beats me to it.

Comment (AlCalzone) [reply packages/config/config/devices/0x0129/ykfcon.json:36]: This manual also mentions the auto relock time of max. 180:
https://products.z-wavealliance.org/ProductManual/File%3Ffolder%3D%26filename%3DMarketCertificationFiles/3211/YaleRealLivingAssureSL_Z-WavePlus_SystemIntegratorsGuide.pdf

Could this be the difference between Yale's Z-Wave Module 1 and 2?

Edit: The manual I found isn't for the keyfree lock though, so maybe they do have two versions of the parameter 🤷🏻‍♂️

**Review by blhoward2 — APPROVED** [review]

**Review by AlCalzone — APPROVED** [review]
We'll clean up the template in another PR

**Pull request merged**